content/applications/inventory_and_mrp/manufacturing/basic_setup/mo_costs.rst

```diff
@@ -0,0 +1,201 @@
+navigate to :menuselection:`Inventory app --> Products --> Products`, and select a product.
+
+The manufacturing cost of the product is displayed per unit of measure in the :guilabel:`Cost`
+field. The value continues to update as the costs of additional |MOs| are factored into the average
+cost.
+
+To the right of the :guilabel:`Cost` field is a :guilabel:`Compute Price from BoM` button. Click
+this button to reset the cost of the product to the expected cost, which only takes into account the
+components and operations listed on the |BoM|.
+
+Be aware that clicking :guilabel:`Compute Price from BoM` does not set the price permanently. The
+cost continues to update based on the average of the |BoM| price and the real cost of any future
+|MOs|.
+
+.. admonition:: Example workflow: manufacturing cost
+   :class: alert alert-success
+
+   Golf product manufacturer *Fairway Fields* produces a variety of golf products, include an indoor
+   *putting green*. They have configured a |BoM| for the putting green so that Odoo automatically
+   calculates the manufacturing cost of each putting green |MO|.
+
+   The |BoM| lists two components:
+
+   - One unit of *green felt*, which costs $20.00.
+   - One unit of a *rubber pad*, which costs $30.00.
+
+   The |BoM| also lists four operations, all of which are carried out at *Assembly Station 1*, which
+   has an hourly operating cost of $30.00:
+
+   - *Cut felt*, which has a default duration of seven minutes, for a total cost of $3.50.
+   - *Cut rubber pad*, which has a default duration of five minutes, for a total cost of $2.50.
+   - *Attach pad to felt*, which has a default duration of 15 minutes, for a total cost of $7.50.
+   - *Cut holes*, which has a default duration of three minutes, for a total cost of $1.50.
+
+   Altogether, the components required to produce one putting green cost $50.00, and the operations
+   required cost $15.00, for a total manufacturing cost of $65.00. This cost is reflected in the
+   Cost field on the putting green's product form.
+
+   Fairway Fields confirms an |MO| for one putting green. Before manufacturing starts, the |MO|
+   overview lists a cost of `$65.00` in both the |MO| Cost and Real Cost fields.
+
+   .. image:: mo_costs/overview-before.png
+      :align: center
+      :alt: The MO Overview page for one putting green, before production starts.
+
+   Manufacturing begins, and the operations take ten minutes longer than expected, for a total
+   manufacturing time of 40 minutes. This deviation from the |BoM| is reflected on the |MO|
+   overview, which now lists a real cost of $70.00.
+
+   .. image:: mo_costs/overview-during.png
+      :align: center
+      :alt: The MO Overview page for one putting green, during production.
+
+   Once manufacturing is finished, and the |MO| is marked as *Done*, the |MO| overview updates again
+   so the values in the MO Cost and Real Cost columns match, each displaying a value of $70.00.
```

Comment:
The MO Cost and Real Cost can have GUI's because you are showing these field sin the images right below it, so I think it applies.